tutorials/neurips_challenge_quickstart.md

```diff
@@ -0,0 +1,201 @@
+
+**Permitted datasets**
+
+Any open-source dataset is allowed. However, [per competition rules](https://llm-efficiency-challenge.github.io/challenge), datasets that utilize "generated content" from other LLMs are not permitted.
+
+Examples of permitted datasets are the following:
+
+- [Databricks-Dolly-15](https://huggingface.co/datasets/databricks/databricks-dolly-15k)
+- [OpenAssistant Conversations Dataset (oasst1)](https://huggingface.co/datasets/OpenAssistant/oasst1)
+- [The Flan Collection](https://github.com/google-research/FLAN/tree/main/flan/v2)
+
+It is allowed to create your own datasets if it is made publicly accessible under an open-source license.
+
+Helpful competition rules relevant to the dataset choice:
+
+- The maximum prompt/completion length the models are expected to handle is 2048 tokens. 
+- The evaluation will be on English texts only.
+
+**Submission deadline**
+
+- October 15, 2023 ([Please check](https://llm-efficiency-challenge.github.io/dates) official website in case of updates.)
+
+&nbsp;
+
+## Lit-GPT Setup
+
+Use the following steps to set up the Lit-GPT repository on your machine.
+
+
+1. Install PyTorch 2.1 (until 2.1 is officially released, you need to install the nightly release):
+
+```bash
+pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre 'torch>=2.1.0dev'
+```
+
+2. Clone the repository:
+
+```bash
+git clone https://github.com/Lightning-AI/lit-gpt.git
+```
+
+
+3. Install the dependencies plus some optional ones:
+
+```bash
+cd lit-gpt
+pip install -r requirements.txt tokenizers sentencepiece
+```
+
+
+
+&nbsp;
+
+## Downloading Model Checkpoints
+
+This section explains how to download the StableLM 3B Base model, the smallest model supported in Lit-GPT. (The downloaded and converted checkpoints will occupy approximately 28 Gb of disk space.)
+
+```
+pip install huggingface_hub
+
+python scripts/download.py \
+  --repo_id stabilityai/stablelm-base-alpha-3b
+
+python scripts/convert_hf_checkpoint.py \
+  --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b
+```
+
+While StableLM 3B Base is useful as a first starter model to set things up, you may want to use the more capable Falcon 7B or Llama 2 7B/13B models later. See the [`download_*`](https://github.com/Lightning-AI/lit-gpt/tree/main/tutorials) tutorials in Lit-GPT to download other model checkpoints.
+
+After downloading and converting the model checkpoint, you can test the model via the following command: 
+
+```
+python generate/base.py \
+  --prompt "LLM efficiency competitions are fun, because" \
+  --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b
+```
+
+&nbsp;
+
+## Downloading and Preparing Datasets 
+
+The following command will download and preprocess the Dolly15k dataset for the StableLM 3B Base model:
+
+```python
+python scripts/prepare_dolly.py \
+--checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b \
+--destination_path data/dolly-stablelm3b \
+--max_seq_length 2048
+```
```

Comment:
```suggestion
```bash
python scripts/prepare_dolly.py \
  --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b \
  --destination_path data/dolly-stablelm3b \
  --max_seq_length 2048
```
```

Reply:
Ah yes, I added this everywhere consistently. Thanks!